nucliadb/src/nucliadb/search/settings.py

```diff
@@ -36,6 +36,11 @@ class Settings(DriverSettings):
         title="Slow node query log threshold",
         description="The threshold in seconds for logging slow node queries",
     )
+    prequeries_max_parallel: int = Field(
```

Comment:
Just wondering, should we implement a semaphore for search in general? I feel we have a lot of semaphores with different numbers per search and it's kind of difficult to know how many asyncio tasks a search is spawning

Reply:
IMO it's fine to have different semaphores. Each task is different in nature: it's not the same querying to pinecone than fetching metadata from postgres, so I think it does make sense to have different settings.

Reply:
I was thinking more about the overall number of tasks a search can create. Anyway, not related with the PR